acceptance/Rakefile

```diff
@@ -85,7 +85,7 @@ def beaker_test(mode = :packages, options = {})
   if mode == :git
     # Build up project git urls based on git server and fork env variables or defaults
     final_options[:install].map! do |install|
-      if md = /^(\w+)#(\w+)$/.match(install)
+      if md = /^(\w+)#([\d.\w]+)$/.match(install)
```

Comment:
> Will this continue to work if we move back to stable?

Yep.

```
irb(main):001:0> /^(\w+)#([\d.\w]+)$/.match('facter#stable')
=> #<MatchData "facter#stable" 1:"facter" 2:"stable">
```

> Can we make a regex that'll match any branch name?

Based on https://www.kernel.org/pub/software/scm/git/docs/git-check-ref-format.html, we will incorrectly match some things, e.g. `..`, `foo.`, and won't match when we should, e.g. `a+b`, `foo/bar`. The authoritative way seems to be to shell out to `git check-ref-format --branch <name>`, but that seems like overkill.